src/uct/base/uct_iface.c

```diff
@@ -194,6 +194,40 @@ static ucs_status_t uct_base_iface_flush(uct_iface_h tl_iface)
     return UCS_OK;
 }
 
+/*
+static ucs_status_t uct_iface_nop_iface_arg(uct_iface_h tl_iface)
+{
+    return tl_iface->status;
+}
+
+static ucs_status_t uct_iface_nop_ep_arg(uct_ep_h ep)
+{
+    return ep->iface->status;
+}
+*/
+
+void uct_base_iface_set_error(uct_iface_h tl_iface, ucs_status_t status)
+{
+    /*
+    void **func_iter;
+    tl_iface->status = status;
+    */
+
+    /* Overwrite functions with interface as first argument */
+    /*for (func_iter = (void**)&tl_iface->ops;
```

Comment:
This is a reference for future implementation, complementing the comment. I don't mind removing it, but I think it clarifies the functionality I'm suggesting for set_error().